ribs/emitters/rankers.py

```diff
@@ -53,7 +53,9 @@
     rng (numpy.random.Generator): A random number generator.
 
 Returns:
-    Indices representing a ranking of the solutions
+    tuple(numpy.ndarray, numpy.ndarray): Indices representing a ranking of the
+    solutions and a ndarray of the metrics that this ranker used to rank
```

Comment:
Also list the dimensions of each array

Reply:
It seems the ranking values may either be 1D (shape `(n,)`) or 2D (shape `(n, x)`) which is okay as long as our code handles it